### PR TITLE
Add release notes configuration

### PR DESCRIPTION
**Pull request by oyvindeide**

### CODE REVIEW

File: .github/release.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        - enhancement
+    - title: Bug Fixes
+      labels:
+        - bug
+    - title: Other Changes
+      labels:
+        - "*"
+    - title: ert3
+      labels:
+        - ert3
```
Comment (sondreso) [inline .github/release.yml:17]: Does this work like an if statement, or are items added to all matching cases?

Comment (oyvindeide) [reply .github/release.yml:17]: I need to move `ert3` up a level, as the `*` is:

> Use * as a catch-all for pull requests that didn't match any of the previous categories.

From the docs it looks like an if statement.

**Comment by oyvindeide** [comment]
test ert please

**Review by sondreso — APPROVED** [review]
Very nice change :+1:

**Pull request merged**